added notes for 1.23.0 release (#896)
rancher/terraform-provider-rancher2: CHANGELOG.md

## 1.23.0 (March 31, 2022)

FEATURES:

> * The Harvester node driver is now supported. https://github.com/rancher/terraform-provider-rancher2/pull/861

Reviewer: ```suggestion
* The Harvester node driver is now supported. See [#861](https://github.com/rancher/terraform-provider-rancher2/pull/861).
```

> * Added support for KEv2 cluster certificate rotation. https://github.com/rancher/terraform-provider-rancher2/pull/882

Reviewer: ```suggestion
* Added support for KEv2 cluster certificate rotation. See [#882](https://github.com/rancher/terraform-provider-rancher2/pull/882).
```

> * Added support for node pool health checks on KEv2 clusters. https://github.com/rancher/terraform-provider-rancher2/pull/889

Reviewer: ```suggestion
* Added support for node pool health checks on KEv2 clusters. See [#889](https://github.com/rancher/terraform-provider-rancher2/pull/889).
```

ENHANCEMENTS:

> * `hetzner_node_driver` Now supports specifying `server_labels`. https://github.com/rancher/terraform-provider-rancher2/issues851

Reviewer: ```suggestion
* `hetzner_node_driver` now supports specifying `server_labels`. See [#851](https://github.com/rancher/terraform-provider-rancher2/pull/851).
```

> * `drain_before_delete` Can now be specified on `rancher2_cluster_v2` resources. https://github.com/rancher/terraform-provider-rancher2/issues890

Reviewer: ```suggestion
* `drain_before_delete` can now be specified on `rancher2_cluster_v2` resources. See [#890](https://github.com/rancher/terraform-provider-rancher2/pull/890).
```

> * Added support to import `rancher2_cloud_credential` resources

Reviewer: ```suggestion
* Added support for `rancher2_cloud_credential` imported resources.
```

> * Added support to `rancher2_node_template` resource for server labels in hetzner_config, https://github.com/rancher/terraform-provider-rancher2/issues/657

Reviewer: ```suggestion
* Added `rancher2_node_template` label for server resources in `hetzner_config`. See [#657](https://github.com/rancher/terraform-provider-rancher2/issues/657).
```

BUG FIXES:

> * Fixed imported KEv2 clusters having improper changes on import. https://github.com/rancher/rancher/issues/36128

Reviewer: ```suggestion
* Importing KEv2 clusters with the Rancher client have their config correctly rewritten. See [#36128](https://github.com/rancher/rancher/issues/36128).
```

> * Fix race condition for empty kubeconfig generation for `rancher2_cluster_sync` resource, https://github.com/rancher/terraform-provider-rancher2/issues/849

Reviewer: ```suggestion
* Fix race condition for empty kubeconfig generation for `rancher2_cluster_sync` resource. See [#849](https://github.com/rancher/terraform-provider-rancher2/issues/849).
```
